fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1978,6 +1960,27 @@ protected void computeFragmentHosts() throws Exception {
         }
     }
 
+    private int findMaxParallelFragmentIndex(PlanFragment fragment) {
+        Preconditions.checkState(!fragment.getChildren().isEmpty(), "fragment has no children");
+
+        // exclude broadcast join right side's child fragments
+        List<PlanFragment> childFragmentCandidates = fragment.getChildren().stream()
+                .filter(e -> e.getOutputPartition() != DataPartition.UNPARTITIONED)
+                .collect(Collectors.toList());
```

Comment:
for sort and limit, the fragment will have one child and its output partition is UNPARTITIONED. so we should check `childFragmentCandidates.isEmpty()` here to avoid return 0

Reply:
no problem even just return 0?